protocol/x/revshare/keeper/revshare.go

```diff
@@ -124,15 +125,17 @@ func (k Keeper) SetUnconditionalRevShareConfigParams(ctx sdk.Context, config typ
 }
 
 // ValidateRevShareSafety roughly checks if the total rev share is valid using the formula below
-// highest_affiliate_taker_share + sum(unconditional_rev_shares) + market_mapper_rev_share < 100%
-// Note: this is just an estimate as affiliate rev share is based on taker fees, while
-// the rest of the rev share is based on net fees.
-// TODO(OTE-788): Revisit this formula to ensure accuracy.
+// highest_affiliate_taker_share * (highest_taker_fee_share_ppm /
```

Comment:
- `(lowest_maker_fee_share_ppm + highest_taker_fee_share_ppm)` I think you mean `maker_fee` and `taker_fee` here, instead of rev share
- Since maker_rebate is negative, I think you want `highest_affiliate_taker_share * lowest_taker_fee / (lowest_maker_fee + lowest_taker_feee)` which gives you the highest possible effect of affiliate rev share on net fee

Reply:
I agree... just realized that lowest taker fee and lower maker fee would also probably be in the same tier